config/development.js

```diff
@@ -1,5 +1,10 @@
 'use strict';
 
 module.exports = {
-  PORT: 3000
+  PORT: 3000,
+  DB_HOST: 'localhost',
+  DB_NAME: 'sfmovies_development',
+  DB_PASSWORD: '',
+  DB_PORT: 5432,
+  DB_USER: 'sfmovies_user'
```

Comment:
Nit: for config properties, we like to try to keep them alphabetized because when you have ~50 of them, it becomes difficult to find the one you want otherwise.